lib/decidim/cleaner/version.rb

```diff
@@ -5,11 +5,11 @@ module Decidim
   # This holds the decidim-meetings version.
   module Cleaner
     def self.version
-      "1.0.1"
+      "4.0.0"
```

Comment:
```suggestion
      "4.1.0"
```

4.0.0 has already been released